lib/tasks/mastodon.rake

```diff
@@ -33,6 +33,36 @@ namespace :mastodon do
     end
   end
 
+  desc 'Add a user by providing their email, username and initial password.' \
+       'The user will receive a confirmation email, then must reset their password before logging in.'
+  task add_user: :environment do
+    print 'Enter email: '
+    email = STDIN.gets.chomp
+
+    print 'Enter username: '
+    username = STDIN.gets.chomp
+
+    print 'Create user and send them confirmation mail (y/n): '
+    confirm = STDIN.gets.chomp
+    puts
+
+    if confirm == 'y'
```

Comment:
I mean, it's kinda more common to have these as `Y/n`, I'm not sure why but it would look more right here.